.changeset/long-keys-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": minor
+---
+
+Changed trigger to workflow_dispatch and switched from issues to pr as a source of metrics
```

Comment:
suggestion: could you provided a description that has more context and allows to understand applied changes without checking the files changed